admin/monitoring/alerts.md

```diff
@@ -0,0 +1,172 @@
+   This creates a new folder within which you can create dashboards as you need.
+
+### 4. Create your first alert
+
+Let's create our first alert! As an example, we'll create an alert
+for when any user is over 80% of their total usage quota.
+
+```{note}
+[PromQL](https://prometheus.io/docs/prometheus/latest/querying/basics/)
+is a query language used for looking at time series metrics, performing
+efficient calculations on them and querying them for visualization with
+Grafana. You'll need to learn *some* promql to be able to fully use the
+Grafana features for visualizations, but you can get along far by copy
+pasting some and tweaking them.
+```
+
+1. Select "Alerting -> Alert Rules" in the left sidebar.
+2. Click the "New alert rule" button on the top right
+3. Enter a descriptive name for this rule, like "Users approaching home directory limit"
+4. Under "Define query and alert condition", you'll have a space to
+   enter a PromQl query.
+   You can play with different promql queries here, but for our use case,
+   use `max(dirsize_total_size_bytes) by (directory) / max(dirsize_hard_limit_bytes) by (directory)`.
+
+   This query calculates the % of home directory size for each user of the
+   total limit available to them, as a fraction between 0 and 1.
+
+   You can press the "Run Queries" button to preview the data
+   that comes from running this query. You'll see one entry
+   for each user, with the value being the % of their limits they
+   have used so far.
+5. Under "Alert condition", specify "0.8" for the text box so it
+   reads "When query is above 0.8". Press the "Preview alert rule
+   condition" button to see what users this alert would fire for.
+   Adjust the % at which you'd like alerts by changing this value
+   from 0.8 to whatever feels convenient for your community.
+6. Under "Add folder and labels", select the folder we created
+   earlier ("Community Maintained Dashboards") so we can keep
+   our alerts organized.
+7. Under "Set evaluation behavior", select "New evaluation group".
+   Give it the name "default", and hit "Create". You'll only
+   have to do this once - for future alerts, you can simply select
+   this group from the dropdown. This tells grafana to evaluate
+   the rule every minute and alert if it sees issues.
+8. Under "Configure notifications", select the slack contact point
+   you created in the earlier step.
+9. Expand the "Muting, grouping and timings (optional)" section and toggle
+   "Override grouping" to be on. By default, Grafana will generate *one* alert
+   for *all* users that are over the limit. This is unwieldy,
+   and we want one alert per user.
+
+   To do so, add `directory` as another item to the dropdown
+   describing various things each alert will be grouped by. We
+   add `directory` because the Prometheus metric uses `directory`
+   to refer to each user's home directory, and you can see
+   it in the preview of the alert if you scroll up.
```

Comment:
I also added a link to the Home Directory Usage Dashboard with the "Link Dashboard and Panel" button, which seems like a useful thing to be able to click on in the Slack alert.